src/Udgam/components/main-page/main-page.js

```diff
@@ -1,15 +1,19 @@
 import React, { useState, useEffect } from 'react';
 import { motion } from 'framer-motion';
+import muskangupta from './landing-images/muskangupta.jpg';
+import amansingh from './landing-images/amansingh.jpg';
+import yogyatasingh from './landing-images/yogyatasingh.jpg';
+import raghavagarwal from './landing-images/raghavagarwal.jpg';
+import khushikapoor from './landing-images/khushikapoor.jpg';
+import kuldeep from './landing-images/kuldeep.jpg';
+import priyanshu from './landing-images/priyanshu.jpeg';
+import sudarshanyadav from './landing-images/sudarshanyadav.jpg';
+import vaibhavsinha from './landing-images/vaibhavsinha.jpg';
 
 import './main-page.css';
 function UDGmain() {
-  const arrimg = [
-    'https://images.unsplash.com/photo-1620336655055-088d06e36bf0?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8Mzh8fHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=500&q=60',
-    'https://images.unsplash.com/photo-1485237254814-0003b25e5672?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8MTF8fHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=500&q=60',
-    'https://images.unsplash.com/photo-1519638399535-1b036603ac77?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8MXx8fGVufDB8fHx8&auto=format&fit=crop&w=500&q=60',
-    'https://images.unsplash.com/photo-1581833971358-2c8b550f87b3?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8Mnx8fGVufDB8fHx8&auto=format&fit=crop&w=500&q=60'
-  ];
-  const names = ['kumar sonu', 'athar', 'nikhil', 'dhruv'];
+  const arrimg = [muskangupta, amansingh, yogyatasingh, raghavagarwal, khushikapoor, kuldeep, priyanshu, sudarshanyadav, vaibhavsinha];
+  const names = ['Muskan Gupta', 'Aman Singh', 'Yogyata Singh', 'Raghav Agrawal', 'Khushi Kapoor', 'Kuldeep', 'Priyanshu', 'Sudarshan Yadav', 'Vaibhav Sinha'];
```

Comment:
Make it "Captured By" instead of clicked by.
First letter of all names should by capital and change its font.